.github/dependabot.yml

```diff
@@ -7,6 +7,8 @@ updates:
     directory: '/'
     schedule:
       interval: weekly
+      day: tuesday
+      time: 09:00
```

Comment:
You mentioned it in the description, but I don't think we need to do this for the design system.  It's good to have this as a point of reference for when we do this on other repos that deploy to production on a merge to main

Reply:
Now removed